libs/actions/common/PythonRunner/pythonrunner.py

```diff
@@ -0,0 +1,180 @@
+import os
+import sys
+import tempfile
+import functools
+import subprocess
+from loguru import logger
+
+
+def _temp_dir_decorator(func: callable):
```

Comment:
@pisces860225: This decorator appears to be unused. Do you think it would be appropriate to add this function here?

Reply:
This part was code left over from debugging and has been removed.